gateway-provider-security-jwt/src/main/java/org/apache/knox/gateway/provider/federation/jwt/filter/JWTFederationFilter.java

```diff
@@ -110,7 +114,7 @@ public void init( FilterConfig filterConfig ) throws ServletException {
     //  JWKSUrl
     String oidcjwksurl = filterConfig.getInitParameter(JWKS_URL);
```

Comment:
Yeah, I thought about that but we have other provisioning tools that are using this config. So depreciating it would mean changing a lot of supporting code. But, I would introduce `knox.token.jwks.urls` property as well.